doc/tutorials/acrn_configurator_tool.rst

```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
       :align: center
```

Comment:
Line 177 or 184 .  "when you need to iterate your"   I recommend changing "iterate" to "change".  I didn't quite understand what iterate meant in that context and change seems more appropriate.